src/style/app.module.css

```diff
@@ -639,17 +683,52 @@
 
 .regularBtn:is(:hover, :active) {
   background-color: #286fe0 !important;
+  color: var(--dark-grey-font);
```

Comment:
Use variable names that describe the function of the color and not the color itself. This will help with adding a Dark Mode feature.
- For example `--search-button-bg` is much better than `--light-blue`